tests/datavolume_test.go

```diff
@@ -2486,7 +2484,8 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		},
 			Entry("[test_id:9922]the storage class is created", testScName, createStorageClass),
 			Entry("[test_id:9924]PV with the SC name is created", testScName, createPV),
-			Entry("[test_id:9925]PV with the SC name (\"\" blank) is created", "", createPV),
+			//FIXME
```

Comment:
Assuming you are working on this still.

Reply:
I'll simply remove the PVC webhook rendering label for this test and revert it to the original code. It tests DV conditions update during rendering, which is irrelevant when using the PVC webhook rendering.